tests/gold_tests/autest-site/microserver.test.ext

```diff
@@ -144,6 +131,39 @@ def makeHeader(self, requestString, **kwargs):
     return headerStr
 
 
+def uServerUpAndRunning(host, port, isSsl):
+    plain_sock = socket.socket(socket.AF_INET)
+    sock = ssl.wrap_socket(plain_sock) if isSsl else plain_sock
```

Comment:
Why not `plain_sock = ssl.wrap_socket(plain_sock) if isSsl;`?

Reply:
Sure I can make this change.

Reply:
* Actually, python does not allow this kind of ternary syntax. You need an else.
* If the socket is wrapped than it is no longer "plain." 

I think maybe I should leave it?

Reply:
Yeah, if Python requires the `else` you can't really improve it.

Reply:
I am not sure what you are saying here. test ? a : b in C++ is A if test else B
so 
`sock = ssl.wrap_socket(plain_sock) if isSsl else socket.socket(socket.AF_INET)
`

Reply:
I am not sure it would make sense to create a second socket in addition to `plain_sock` in the case `isSsl` is not true.

But I'm willing to change to whatever style folks want.